components/global/history_buttons/history_buttons.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import styled from 'styled-components';
+import IconButton from '@mattermost/compass-components/components/icon-button';
+
+import {trackEvent} from 'actions/telemetry_actions';
+import * as Utils from 'utils/utils';
+import {isDesktopApp} from 'utils/user_agent';
+import {browserHistory} from 'utils/browser_history';
+
+const HistoryButtonsContainer = styled.nav`
+    display: flex;
+    align-items: center;
+`;
+
+const HistoryButtons = (): JSX.Element | null => {
+    if (!isDesktopApp()) {
+        // need to re-enable this
+        // return null;
+    }
+
+    const goBack = () => {
+        trackEvent('ui', 'ui_history_back');
+        browserHistory.goBack();
```

Comment:
I think the correct way to do this now is with the `useHistory` hook: https://reactrouter.com/web/api/Hooks/usehistory

Reply:
will have to discuss with desktop app team as well.

Reply:
This is porting existing functionality. If we want to handle differently moving forward, we should open a ticket and deal with it in a separate PR.